core/src/main/java/org/apache/spark/shuffle/unsafe/UnsafeShuffleWriter.java

```diff
@@ -125,6 +129,7 @@ public UnsafeShuffleWriter(
     taskContext.taskMetrics().shuffleWriteMetrics_$eq(Option.apply(writeMetrics));
     this.taskContext = taskContext;
     this.sparkConf = sparkConf;
+    this.allowSpillMove = sparkConf.getBoolean("spark.shuffle.unsafe.testing.allowSpillMove", true);
```

Comment:
I think at the same time you were adding this comment, I put in a comment, but on the variable declaration: https://github.com/apache/spark/pull/6648/files#diff-642ce9f439435408382c3ac3b5c5e0a0R99